### PR TITLE
Release [v1.0.0]

### PR DESCRIPTION
**Pull request by rickybasse**
## Release [v1.0.0]

This PR contains: 

* Terraform code for creating a Scheduled Fargate Task in AWS ECS
* Terraform code with a complete usage example 
* Github Actions workflow with linting and validation
* Documentation

TODO: 
- [x] Wait for GitHub to release Actions for general use [13th November]

### CODE REVIEW

File: CODE_OF_CONDACT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```
Comment (Dimwest) [inline CODE_OF_CONDACT.md]: typo in the filename

File: examples/complete/task/task-definition.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        "logConfiguration": {
+            "logDriver": "awslogs",
+            "options": {
+                "awslogs-group": "/ecs/${image_name}-logs",
+                "awslogs-region": "${region}",
+                "awslogs-stream-prefix": "ecs"
+            }
+        }
+    }
+]
```
Comment (Dimwest) [inline examples/complete/task/task-definition.json]: newline

File: version [hunk 3]
```diff
@@ -0,0 +1 @@
+1.0.0
```
Comment (Dimwest) [inline version]: newline

**Review by Dimwest — CHANGES_REQUESTED** [review]
Almost nothing to say, just minor comments about a typo and newlines

**Comment by unsign3d** [comment]
I put as reviewers the OSS CoP as a FYI

**Comment by unsign3d** [comment]
but from the OSS point of view, it seems ok

**Review by unsign3d — APPROVED** [review]

**Review by dzeno — APPROVED** [review]

**Pull request merged**